index.js

```diff
@@ -4,10 +4,12 @@ const PluginError = require('plugin-error');
 
 const PLUGIN_NAME = 'gulp-append-prepend';
 
-function filesGetContents(filepaths){
-    if (!(filepaths instanceof Array)) {
-        filepaths = [filepaths];
-    }
+function arraify(maybeArr) {
+    return Array.isArray(maybeArr) ? maybeArr : [maybeArr];
+}
```

Comment:
I think `arrayify` is a better name

```suggestion
function arrayify(obj) {
    return Array.isArray(obj) ? obj : [obj];
}
```

Reply:
Sure - seems like "arraify" is the more conventional name as well. 
Also no issue with renaming `maybeArr` as well - just my personal preference.